### PR TITLE
[SYCL][matrix] Add basic bf16 test case for the joint matrix feature

### PR DESCRIPTION
**Pull request by dkhaldi**

### CODE REVIEW

**Comment by romanovvlad** [comment]
@AlexeySotkin Could you please review?

File: SYCL/Matrix/joint_matrix_bf16.cpp [hunk 1]
```diff
@@ -0,0 +1,197 @@
+//==-------- joint_matrix_bf16.cpp  - DPC++ joint_matrix--------------- ----==//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+// REQUIRES: matrix
```
Comment (AlexeySotkin) [inline SYCL/Matrix/joint_matrix_bf16.cpp:8]: I think we also need some code in `SYCL/lit.cfg.py` to add "marix" to `config.available_features`

**Comment by AlexeySotkin** [comment]
In general, looks good to me but I'd like to see results of the test.

File: SYCL/Matrix/joint_matrix_bf16.cpp (hunk 1 above)
Comment (yuxianch) [reply SYCL/Matrix/joint_matrix_bf16.cpp:8]: The change is made in https://github.com/intel/llvm-test-suite/pull/385

**Review by AlexeySotkin — APPROVED** [review]

**Review by romanovvlad — APPROVED** [review]

**Pull request merged**